ci_framework/roles/set_openstack_containers/molecule/default/converge.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  roles:
+    - role: "set_openstack_containers"
```

Comment:
for some reason it's failing now on not finding `cifmw_openshift_kubeconfig` parameter. You probably want to pass some fake one - unless you're consuming CRC and therefore can use the CRC one, for instance as it's done in the openshift_setup molecule tests:
`cifmw_openshift_kubeconfig: "{{ ansible_user_dir }}/.crc/machines/crc/kubeconfig"`

(source: roles/openshift_setup/molecule/default/converge.yml)

Reply:
ah i missed to update the nodeset, will update once current run on testproject finishes.